api/fixtures/example.go

```diff
@@ -3,6 +3,7 @@ package fixtures
 import (
 	"crypto/rand"
 	"fmt"
+	"k8s.io/apimachinery/pkg/util/intstr"
```

Comment:
move this in the below section of k8s imports

Reply:
moved